packages/TimePicker/package.json

```diff
@@ -17,7 +17,9 @@
     "@babel/runtime-corejs2": "^7.3.1",
     "@paprika/helpers": "^0.2.13",
     "@paprika/icon": "^0.3.3",
-    "@paprika/input": "^0.3.0",
+    "@paprika/input": "0.3.0",
+    "@paprika/l10n": "0.4.5",
+    "@paprika/popover": "0.3.29",
```

Comment:
Is there any reason for using fixed versions?

Reply:
There is not, so I'll adjust that quickly